### PR TITLE
EFS changes and New runtime changes

### PR DESCRIPTION
**Pull request by rajeevr2715**

### CODE REVIEW

File: jenkins/Dockerfile [hunk 1]
```diff
@@ -32,20 +32,21 @@ wget https://aka.ms/downloadazcopy-v10-linux -O /opt/azurecopy.tar.gz && tar xzv
...
 COPY jenkins/plugins.txt /usr/share/jenkins/ref/plugins.txt
 RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh && /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt
 
 ADD jenkins/files/xmls.tar /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/encrypt.groovy /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/credentials.xml /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/nodeMonitors.xml /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/config.xml /var/jenkins_home/
-COPY --chown=jenkins:jenkins jenkins/files/scriptApproval.xml /var/jenkins_home/
+COPY  jenkins/files/copy_reference_file.log /usr/share/jenkins/ref/copy_reference_file.log
```
Comment (raghits) [inline jenkins/Dockerfile:43]: Why are we changing this?

File: jenkins/Dockerfile [hunk 2]
```diff
@@ -32,20 +32,21 @@ wget https://aka.ms/downloadazcopy-v10-linux -O /opt/azurecopy.tar.gz && tar xzv
...
+COPY  jenkins/files/encrypt.groovy /usr/share/jenkins/ref/encrypt.groovy
```
Comment (raghits) [inline jenkins/Dockerfile:44]: Why are we changing the following 5 lines.?

**Review by raghits — COMMENTED** [review]
See comments.

File: jenkins/Dockerfile (hunk 2 above)
Comment (rajeevr2715) [reply jenkins/Dockerfile:44]: During the ECS - EFS integration the Jenkins image was running into issues during spin up. So I had to make these changes to work.

File: jenkins/Dockerfile (hunk 1 above)
Comment (rajeevr2715) [reply jenkins/Dockerfile:43]: During the ECS - EFS integration the Jenkins image was running into issues during spin up. So I had to make these changes to work.

**Comment by rajeevr2715** [comment]
Done

**Review by raghits — APPROVED** [review]
LGTM

**Pull request merged**